api/v4/source/system.yaml

```diff
@@ -47,6 +47,11 @@
 
         __Minimum server version__: 6.5
 
+        If "keep_rest_semantics" is set to true in the query, the endpoint will not return
```

Comment:
`keep_rest_semantics` seems a bit misleading to me, as the name indicates that the semantics have changed recently. What about `use_rest_semantics`?

Reply:
Makes sense.